recipes/geocube/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.0.2" %}
+
+package:
+  name: geocube
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/g/geocube/geocube-{{ version }}.tar.gz
+  sha256: ef74e677510ab9ef17bb855d65c187fbd50098a5697effdd4749b9cd5c516d79
+
+build:
+  number: 0
+  noarch: python
+  # entry_points:
```

Comment:
Is this commented out for a reason? (It's the cause of the build failure)

Reply:
> Any updates on this? Anything needed from my end?

Not really. I will leave the CLI stuff commented out until we get a new release. Thanks!

Reply:
> I'm 90% sure they're already in this release as this Windows failure is normally caused by not declaring entrypoints.

@chrisburr the entrypoint was declared in the setup.py but the module does not exist. That was already updated upstream and we are waiting on a new release but it is safe to merge the `noarch` now.